omero/omero-firewall.yml

```diff
@@ -0,0 +1,62 @@
+# Setup up iptables firewall on OMERO servers
+
+# TODO: change to prod-omero, prod-omero-web
+- hosts: pub-omero.openmicroscopy.org
+#- hosts: prod-omero, prod-omero-web
+
+  roles:
+
+  - role: openmicroscopy.iptables-raw
+
+  tasks:
+
+  # Allow:
+  # - all established/related in/out
+  # - all internal localhost connections
+  # - ICMP echo (ping)
+  # - ssh incoming connections
+  - name: Iptables ssh and related
+    become: yes
+    iptables_raw:
+      name: ssh_and_established
+      keep_unmanaged: no
+      rules: |
+        -A INPUT -m state --state ESTABLISHED,RELATED -j ACCEPT
+        -A OUTPUT -m state --state ESTABLISHED,RELATED -j ACCEPT
+        -A INPUT -i lo -j ACCEPT
+        -A INPUT -p icmp --icmp-type echo-request -j ACCEPT
+        -A INPUT -p tcp -m tcp --dport 22 -j ACCEPT
+      state: present
+      # Highest priority
+      weight: 0
+
+  # Use a low priority REJECT rule so that clients can detect when
+  # they've been rejected
+  # The alternative of setting a default DROP policy will leave them
+  # hanging until they timeout, though this may be preferable for public
+  # servers:
+  # http://www.chiark.greenend.org.uk/~peterb/network/drop-vs-reject
+  - name: Iptables default
+    become: yes
+    iptables_raw:
+      name: default_reject
+      rules: |
+        -A INPUT -j REJECT
+        -A FORWARD -j REJECT
+        -A OUTPUT -j ACCEPT
+      state: present
+      # Lowest priority
+      weight: 99
+
+  # All other ports that allow incoming connections
+  - name: Iptables OME ports
+    become: yes
+    iptables_raw:
+      name: ome_ports
+      rules: |
+        -A INPUT -p tcp -m tcp --dport 80 -j ACCEPT
+        -A INPUT -p tcp -m tcp --dport 443 -j ACCEPT
+        -A INPUT -p tcp -m tcp --dport 4063 -j ACCEPT
+        -A INPUT -p tcp -m tcp --dport 4064 -j ACCEPT
+        -A INPUT -p tcp -m tcp --dport 6556 -j ACCEPT
```

Comment:
the multiports extension's `--dports` might be nice here

Reply:
Thanks, I thought that only worked for port ranges. I've updated it to use multiport, but split by application.